crates/openapi/src/routes/organization.rs

```diff
@@ -24,28 +25,30 @@
 )]
 pub async fn organization_create() {}
 
+#[cfg(any(feature = "v1", feature = "v2"))]
 /// Organization - Retrieve
 ///
 /// Retrieve an existing organization
 #[utoipa::path(
-    post,
+    get,
     path = "/organization/{organization_id}",
     params (("organization_id" = String, Path, description = "The unique identifier for the Organization")),
     responses(
         (status = 200, description = "Organization Created", body =OrganizationResponse),
         (status = 400, description = "Invalid data")
     ),
     tag = "Organization",
-    operation_id = "Create an Organization",
+    operation_id = "Retrive an Organization",
```

Comment:
```suggestion
    operation_id = "Retrieve an Organization",
```

Please re-generate the OpenAPI spec once you make this change.